src/sparsezoo/__init__.py

```diff
@@ -25,3 +26,59 @@
 from .main import *
 from .models.zoo import *
 from .objects import *
+
+from .requests.base import LATEST_PACKAGE_VERSION_URL
+
+import logging
+import requests
+import threading
+import os
+
+_LOGGER = logging.getLogger(__name__)
+
+def _version_check(package_name, package_version, package_integration):
```

Comment:
@kazukageorge let's make this base function public as the other repositories will be expected to call it. As mentioned in the other comment, we'll need to nest this in a separate, appropriate package and then import the callable in here and rename as private so we don't export it from the root as a public function